lang/ff7tk_it.ts

```diff
@@ -5,71 +5,72 @@
     <name>CharEditor</name>
     <message>
         <source>Name</source>
-        <translation>Alfabetico</translation>
+        <translation>Nome</translation>
     </message>
     <message>
         <source>Current HP</source>
-        <translation type="unfinished">PV Attuali</translation>
+        <translation>PV attuali</translation>
     </message>
     <message>
         <source>Base HP</source>
-        <translation type="unfinished">PV fondamento</translation>
+        <translation>PV base</translation>
     </message>
     <message>
         <source>HP</source>
         <translation>PV</translation>
     </message>
     <message>
         <source>Materia and Equipment Bonus</source>
-        <translation type="unfinished">Bonus Materie e Attrezzatura</translation>
+        <translation>Bonus materie e equipaggiamento</translation>
     </message>
     <message>
         <source>Maximum HP</source>
-        <translation type="unfinished">PV massimo</translation>
+        <translation>PV massimi</translation>
     </message>
     <message>
         <source>Current MP</source>
-        <translation type="unfinished">PM attuali</translation>
+        <translation>PM attuali</translation>
     </message>
     <message>
         <source>Base MP</source>
-        <translation type="unfinished">PM fondamento</translation>
+        <translation>PM base</translation>
     </message>
     <message>
         <source>MP</source>
-        <translation type="unfinished">PM</translation>
+        <translation>PM</translation>
     </message>
     <message>
         <source>Maximum MP</source>
-        <translation>Massimo PM</translation>
+        <translation>PM massimi</translation>
     </message>
     <message>
         <source>Level</source>
         <translation>Livello</translation>
     </message>
     <message>
         <source>Kills</source>
-        <translation>uccide</translation>
+        <translation>Uccisioni</translation>
     </message>
     <message>
         <source>Fury</source>
-        <translation type="unfinished">Furia</translation>
+        <translation>Furia</translation>
     </message>
     <message>
         <source>Sadness</source>
-        <translation type="unfinished">Apatia</translation>
+        <translation>Sconforto</translation>
     </message>
     <message>
         <source>Front Row</source>
-        <translation type="unfinished">Prima fila</translation>
+        <translation>Prima fila</translation>
     </message>
     <message>
         <source>Next</source>
-        <translation type="unfinished">Avanti</translation>
+        <translatorcomment>Double check where it&apos;s used</translatorcomment>
```

Comment:
Used in the Character Editor 
Next  - as is the "Amount of Exp to the Next Level"
Its later replaced with "Next: %1" when data is populated.